.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: java
+
+jdk:
+  - oraclejdk8
+
+env:
+  - DB=mysql
+  - DB=psql
+
+services:
+  - mysql
+  - postgresql
+
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -y activemq -qq
+  - sudo ln -s /etc/activemq/instances-available/main /etc/activemq/instances-enabled/main
+  - sudo sed -e 's/<broker /<broker schedulerSupport="true" /' -i /etc/activemq/instances-enabled/main/activemq.xml
+  - sudo service activemq restart
+  - sudo apt-get install -y tomcat7
+  - sudo service tomcat7 stop
+  - echo 'CATALINA_OPTS="-Xms512m -Xmx512m"' | sudo tee --append /usr/share/tomcat7/bin/setenv.sh
+  - sudo chown -R tomcat7:tomcat7 /var/lib/tomcat7/ /usr/share/tomcat7/ /var/log/tomcat7/
+
+before_script:
+  - sudo chmod +x ./travis_before_build.sh
+  - ./travis_before_build.sh
+
+script:
+  - mvn clean install -PIT -q
```

Comment:
This is a default command in "install" phase of build. If we want to change this, we just should add e.g. "install: mvn clean install" to this file. 
Here it is in Travis documentation:
https://docs.travis-ci.com/user/languages/java/#Dependency-Management